packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -202,26 +205,45 @@ export const Table: React.FunctionComponent = () => {
   const [sortDirection, setSortDirection] = useState<'asc' | 'desc'>('asc')
   const [items, setItems] = useState([])
   const [metaData, setMetaData] = useState<MetaData>()
+  const [deleteId, setDeleteId] = useState<string[]>([])
+  const [isModalVisible, setIsModalVisible] = useState(false)
+  console.log(selectedItems)
+  const getData = async () => {
```

Comment:
A practice I use is, I separate unrelated steps with one space. So since the definition of `params` is different from the `fetchTableData` call, I skip a line before putting it. Just to make the code much less crowded. So this is how i'll write this function:


```js
const getData = () => {
    const params = {
      page: pageIndex + 1,
      perPage: pageSize,
      sort: `${sortField}:${sortDirection}`,
      sortField
    }

    const [data, error] = await fetchTableData(params)

    if (!error) {
      setItems(data?.data.data)
      setMetaData(data?.data.meta)
      setLoading(false)
    }

    setLoading(false) // if there is an error so it doesn't load forever
  }
}
```